pkg/util/yaml.go

```diff
@@ -164,6 +168,21 @@ func OverlayYAML(base, overlay string) (string, error) {
 	return string(my), nil
 }
 
+func ReadLayeredYAMLs(filenames []string) (string, error) {
```

Comment:
This is not specific to a command or istioctl. It can be  a common pattern for multiple yaml input files. If the concern is about the test files, I can change it not related to ICPS, just plain yamls.

Reply:
test updated.

Reply:
it's not really about using ICPS but rather that the function is not generally useful but rather specific to a couple of commands.